mobile/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ publish_to: 'none' # Remove this line if you wish to publish to pub.dev
 # Read more about iOS versioning at
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # android test
-version: 2.0.5+20011
+version: 2.0.6+20012
```

Comment:
updating android version

Reply:
just curious about this manual update of the android versions. Could we explore just stating the minimum (starting) version or an approach that does not involve the manual update of versions.

Reply:
unfrortunately play store and app store dont support auto updating. It is the developer to set the version or have it configured in CI/CD @Baalmart